packages/react-native/src/integrations/console.ts

```diff
@@ -1,32 +1,35 @@
 export const INJECT_DEBUG = `
 {
-  const originalConsoleLog = console.log;
-  const originalConsoleError = console.error;
-  const originalConsoleWarn = console.warn;
+  if (!window["webview_bridge_debug_injected"]) {
```

Comment:
```suggestion
  if (window["webview_bridge_debug_injected"]) {
    return;
  }
```

how about use early return  ?

Reply:
and It’s a minor detail, but as shown above, generating a random value and using it can be a good idea.
Although the possibility is low, storing it on the window object can cause name collisions.

 ```ts
import { createRandomId } from "@webview-bridge/utils";

const randomId = createRandomId();

export const INJECT_DEBUG = `
{
  const originalConsoleLog = console.log;
  const originalConsoleError = console.error;
  const originalConsoleWarn = console.warn;
  if (!window["webview_bridge_debug_injected_${randomId}"]) { // <<
```

Reply:
> Although the possibility is low, storing it on the window object can cause name collisions.

I was worried about naming collisions too, but couldn't think of a way around it.
This seems like the perfect solution.

Reply:
What do you think about prefix `webview_bridge_debug_injected_`?
If there is a naming convention within the project, I will follow it.

Reply:
`__bridge_debug_injected_{randomId}` 👍